quickwit-metastore/.env

```diff
@@ -1,4 +1,5 @@
 POSTGRES_USER=quickwit-dev
 POSTGRES_PASSWORD=quickwit-dev
-DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost/quickwit-dev
-TEST_DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost/quickwit-dev
+POSTGRES_DB=quickwit-metastore
+DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost:5432/quickwit-metastore
+TEST_DATABASE_URL=postgres://quickwit-dev:quickwit-dev@localhost:5432/quickwit-metastore
```

Comment:
Why do we need both `DATABASE_URL` and `TEST_DATABASE_URL` if they are the same? I'd rather keep `TEST_DATABASE_URL,` which has a more explicit name.

Reply:
Yes, we don't need the `DATABASE_URL` here. I'll delete it.

Reply:
If this is only for tests then the file location is not ideal

Reply:
@PSeitz 
You're right, it's better to put it in the same place as docker-compose.yml.